### PR TITLE
fix: updates extensionSDK and bumps timeout to 500s

### PR DESCRIPTION
**Pull request by noahamac**
Had to make some updates to get `chattyTimeout` on the ExtensionProvider

Updated broken tests, re-linted

You can test it out here, after `yarn start`
https://app.dev.looker.com/extensions/mp-viz::dev2-data-dictionary/models/covid/explores/covid_combined

### CODE REVIEW

File: src/components/DataDictionary.tsx [hunk 1]
```diff
@@ -90,6 +90,7 @@ export const columns: ColumnDescriptor[] = [
     name: "category",
     label: "Category",
     rowValueDescriptor: "category",
+    // eslint-disable-next-line react/display-name
```
Comment (noahamac) [inline src/components/DataDictionary.tsx:93]: Bleh!

Looks like eslint doesn't like these formatter functions. Another reason to refactor this functionality

File: src/utils/queries.ts [hunk 2]
```diff
@@ -2,9 +2,9 @@ import {
   ILookmlModel,
   ILookmlModelExplore,
   ILookmlModelExploreField
-} from '@looker/sdk/lib/sdk/3.1/models'
+} from '@looker/sdk/lib/3.1/models'
```
Comment (bryans99) [inline src/utils/queries.ts:5]: Should be from 4.0. You are using LookerSDK40

File: src/utils/queries.ts [hunk 3]
```diff
@@ -2,9 +2,9 @@ import {
...
 import { exploreURL } from "./urls"
-import { Looker31SDK as LookerSDK } from '@looker/sdk/lib/sdk/3.1/methods'
+import { Looker31SDK as LookerSDK } from '@looker/sdk/lib/3.1/methods'
```
Comment (bryans99) [inline src/utils/queries.ts:7]: Ditton

File: src/utils/urls.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import {
   ILookmlModelExplore,
   ILookmlModelExploreField
-} from '@looker/sdk/lib/sdk/3.1/models'
+} from '@looker/sdk/lib/3.1/models'
```
Comment (bryans99) [inline src/utils/urls.ts:4]: 4.0

**Review by bryans99 — CHANGES_REQUESTED** [review]
Referencing 3.1 methods.

Smaller PRs! This could have been 2 I think

**Comment by noahamac** [comment]
Resolved by https://github.com/looker-open-source/app-data-dictionary/pull/56

**Pull request closed without merging**